docs/source/getting-started/install.md

```diff
@@ -102,7 +119,7 @@ should not throw an error and show the current running containers.
 
 ## Run a Volto ready Plone Docker container
 
-When you have installed Docker, you can run an standard Plone Docker container with the proper configuration for Volto using the `plone.volto` add'on right away by issuing:
+When you have installed Docker, you can use the official Plone Docker container with the proper configuration for Volto using the `plone.volto` add'on right away by issuing:
 
 ```shell
 docker run -it --rm --name=plone \
```

Comment:
Meh, this one is outdated (see README ones), they use the pip based ones already.

Reply:
I'm not up to date as to which image with which plone version are already using the plone-backend / plone-fontend ones.  :-/